specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/managedDatabases.json

```diff
@@ -0,0 +1,556 @@
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The requested database resource state.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagedDatabaseUpdate"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the database.",
+            "schema": {
+              "$ref": "#/definitions/ManagedDatabase"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ResourceNotFound - The requested resource was not found."
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Updates a managed database with minimal properties": {
+            "$ref": "./examples/ManagedDatabaseUpdateMin.json"
+          },
+          "Updates a managed database with maximal properties": {
+            "$ref": "./examples/ManagedDatabaseUpdateMax.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ManagedDatabaseProperties": {
+      "description": "The managed database's properties.",
+      "type": "object",
+      "properties": {
+        "collation": {
+          "description": "Collation of the managed database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "status": {
+          "description": "Status for the database.",
+          "enum": [
+            "Online",
+            "Offline",
+            "Shutdown",
+            "Creating",
+            "Inaccessible"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ManagedDatabaseStatus",
+            "modelAsString": true
+          }
+        },
+        "creationDate": {
+          "format": "date-time",
+          "description": "Creation date of the database.",
+          "type": "string",
+          "readOnly": true
+        },
+        "pointInTime": {
+          "format": "date-time",
+          "description": "Conditional. If createMode is PointInTimeRestore, this value is required. Specifies the point in time (ISO8601 format) of the source database that will be restored to create the new database.",
+          "type": "string",
+          "readOnly": true
+        },
+        "defaultSecondaryLocation": {
+          "description": "Geo pared region.",
```

Comment:
>pared [](start = 30, length = 5)

Maybe a typo? What does this mean?

Reply:
Yes, it is supposed to be "Geo paired region". Thanks!